src/app/home/home.component.ts

```diff
@@ -28,7 +34,29 @@ export class HomeComponent implements OnDestroy {
     }
   }
 
+  ngOnInit() {
+    this.videoQualitySelect = document.querySelector('#video-quality')! as HTMLSelectElement;
+    this.videoCameraSelect = document.querySelector('#video-camera')! as HTMLSelectElement;
+    this.videoQualitySelect.value =
+      localStorage.getItem('defaultQuality') || '480';
+    this.videoCameraSelect.value =
+      localStorage.getItem('defaultCamera') || 'user';
+  }
+
   ngOnDestroy() {
     this.subsink.unsubscribe();
   }
+
+  onClick() {
+    let quality = this.videoQualitySelect!.options[this.videoQualitySelect!.selectedIndex].value;
+    let camera = this.videoCameraSelect!.options[this.videoCameraSelect!.selectedIndex].value;
+    localStorage.setItem('defaultQuality', quality);
+    localStorage.setItem('defaultCamera', camera);
```

Comment:
I think this can happen in the `selectionChange` events of the `mat-select`s.

Reply:
I now read the local storage directly as suggested below. If I only do this on change, it would mean I would have to handle the defaults everywhere consistently. That way is safer to make sure what we show to the user is actually what will be used.